Ran in order:

```
gh pr checkout 10417
git clean . -Xdf
dotnet build
```

Received errors:

```
Each package is licensed to you by its owner. NuGet is not responsible for, nor does it grant any licenses to, third-party packages. Some packages may include dependencies which are governed by additional licenses. Follow the package source (feed) URL to determine any dependencies.

Package Manager Console Host Version 6.13.0.46

Type 'get-help NuGet' to see all available NuGet commands.

PM> gh pr checkout 10417
gh : From https://github.com/MudBlazor/MudBlazor
At line:1 char:1
+ gh pr checkout 10417
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (From https://gi...lazor/MudBlazor:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
 * [new ref]             refs/pull/10417/head -> sasscompiler-workaround2
Switched to branch 'sasscompiler-workaround2'
PM> git clean . -Xdf
git : warning: failed to remove .vs/MudBlazor/FileContentIndex/4a924da2-a320-4d18-a8a0-f79b6fd67cc5.vsidx: Invalid argument
At line:1 char:1
+ git clean . -Xdf
+ ~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (warning: failed...nvalid argument:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
warning: failed to remove .vs/MudBlazor/FileContentIndex/5f340477-3355-46fb-8d9b-3cc8cf521c37.vsidx: Invalid argument
warning: failed to remove .vs/MudBlazor/FileContentIndex/699d30a6-3cd8-41bb-84b7-12b9b1eda6e0.vsidx: Invalid argument
warning: failed to remove .vs/MudBlazor/FileContentIndex/6f9fef7f-3fc5-478f-beeb-8054a428a4dd.vsidx: Invalid argument
warning: failed to remove .vs/MudBlazor/FileContentIndex/950d938e-32d8-4568-a044-32f616c18112.vsidx: Invalid argument
Removing .vs/MudBlazor/config
Removing .vs/MudBlazor/DesignTimeBuild
Removing .vs/MudBlazor/v17
Removing .vs/ProjectEvaluation
Removing MudBlazor.Analyzers.TestComponents/bin/
Removing MudBlazor.Analyzers.TestComponents/obj/
Removing MudBlazor.Analyzers/bin/
Removing MudBlazor.Analyzers/obj/
Removing MudBlazor.Docs.Compiler/MudBlazor.min.css
Removing MudBlazor.Docs.Compiler/bin/
Removing MudBlazor.Docs.Compiler/obj/
Removing MudBlazor.Docs.Server/bin/
Removing MudBlazor.Docs.Server/obj/
Removing MudBlazor.Docs.Wasm/bin/
Removing MudBlazor.Docs.Wasm/obj/
Removing MudBlazor.Docs.WasmHost/bin/
Removing MudBlazor.Docs.WasmHost/obj/
Removing MudBlazor.Docs/Models/DocStrings.generated.cs
Removing MudBlazor.Docs/Models/Generated/ApiDocumentation.generated.cs
Removing MudBlazor.Docs/Models/Snippets.generated.cs
Removing MudBlazor.Docs/NewFilesToBuild.txt
Removing MudBlazor.Docs/Pages/CSS Utilities/Borders/Code/
Removing MudBlazor.Docs/Pages/CSS Utilities/Flexbox/Code/
Removing MudBlazor.Docs/Pages/CSS Utilities/Interactivity/Code/
Removing MudBlazor.Docs/Pages/CSS Utilities/Layout/Code/
Removing MudBlazor.Docs/Pages/CSS Utilities/Spacing/Code/
Removing MudBlazor.Docs/Pages/Components/Alert/Code/
Removing MudBlazor.Docs/Pages/Components/AppBar/Code/
Removing MudBlazor.Docs/Pages/Components/Autocomplete/Code/
Removing MudBlazor.Docs/Pages/Components/Avatar/Code/
Removing MudBlazor.Docs/Pages/Components/Badge/Code/
Removing MudBlazor.Docs/Pages/Components/Breadcrumbs/Code/
Removing MudBlazor.Docs/Pages/Components/BreakpointProvider/Code/
Removing MudBlazor.Docs/Pages/Components/Button/Code/
Removing MudBlazor.Docs/Pages/Components/ButtonFAB/Code/
Removing MudBlazor.Docs/Pages/Components/ButtonGroup/Code/
Removing MudBlazor.Docs/Pages/Components/Card/Code/
Removing MudBlazor.Docs/Pages/Components/Carousel/Code/
Removing MudBlazor.Docs/Pages/Components/Charts/Code/
Removing MudBlazor.Docs/Pages/Components/Checkbox/Code/
Removing MudBlazor.Docs/Pages/Components/Chip/Code/
Removing MudBlazor.Docs/Pages/Components/ChipSet/Code/
Removing MudBlazor.Docs/Pages/Components/Collapse/Code/
Removing MudBlazor.Docs/Pages/Components/ColorPicker/Code/
Removing MudBlazor.Docs/Pages/Components/Container/Code/
Removing MudBlazor.Docs/Pages/Components/DataGrid/Code/
Removing MudBlazor.Docs/Pages/Components/DatePicker/Code/
Removing MudBlazor.Docs/Pages/Components/DateRangePicker/Code/
Removing MudBlazor.Docs/Pages/Components/Dialog/Code/
Removing MudBlazor.Docs/Pages/Components/Divider/Code/
Removing MudBlazor.Docs/Pages/Components/Drawer/Code/
Removing MudBlazor.Docs/Pages/Components/DropZone/Code/
Removing MudBlazor.Docs/Pages/Components/Element/Code/
Removing MudBlazor.Docs/Pages/Components/ExpansionPanel/Code/
Removing MudBlazor.Docs/Pages/Components/Field/Code/
Removing MudBlazor.Docs/Pages/Components/FileUpload/Code/
Removing MudBlazor.Docs/Pages/Components/FocusTrap/Code/
Removing MudBlazor.Docs/Pages/Components/Form/Code/
Removing MudBlazor.Docs/Pages/Components/Grid/Code/
Removing MudBlazor.Docs/Pages/Components/Hidden/Code/
Removing MudBlazor.Docs/Pages/Components/Highlighter/Code/
Removing MudBlazor.Docs/Pages/Components/IconButton/Code/
Removing MudBlazor.Docs/Pages/Components/Icons/Code/
Removing MudBlazor.Docs/Pages/Components/Image/Code/
Removing MudBlazor.Docs/Pages/Components/Link/Code/
Removing MudBlazor.Docs/Pages/Components/List/Code/
Removing MudBlazor.Docs/Pages/Components/Menu/Code/
Removing MudBlazor.Docs/Pages/Components/MessageBox/Code/
Removing MudBlazor.Docs/Pages/Components/NavMenu/Code/
Removing MudBlazor.Docs/Pages/Components/NumericField/Code/
Removing MudBlazor.Docs/Pages/Components/Overlay/Code/
Removing MudBlazor.Docs/Pages/Components/Pagination/Code/
Removing MudBlazor.Docs/Pages/Components/Paper/Code/
Removing MudBlazor.Docs/Pages/Components/Popover/Code/
Removing MudBlazor.Docs/Pages/Components/Progress/Code/
Removing MudBlazor.Docs/Pages/Components/Radio/Code/
Removing MudBlazor.Docs/Pages/Components/Rating/Code/
Removing MudBlazor.Docs/Pages/Components/ScrollToTop/Code/
Removing MudBlazor.Docs/Pages/Components/Select/Code/
Removing MudBlazor.Docs/Pages/Components/SimpleTable/Code/
Removing MudBlazor.Docs/Pages/Components/Skeleton/Code/
Removing MudBlazor.Docs/Pages/Components/Slider/Code/
Removing MudBlazor.Docs/Pages/Components/Snackbar/Code/
Removing MudBlazor.Docs/Pages/Components/Spacer/Code/
Removing MudBlazor.Docs/Pages/Components/Sparkline/
Removing MudBlazor.Docs/Pages/Components/Stack/Code/
Removing MudBlazor.Docs/Pages/Components/Stepper/Code/
Removing MudBlazor.Docs/Pages/Components/SwipeArea/Code/
Removing MudBlazor.Docs/Pages/Components/Switch/Code/
Removing MudBlazor.Docs/Pages/Components/Table/Code/
Removing MudBlazor.Docs/Pages/Components/Tabs/Code/
Removing MudBlazor.Docs/Pages/Components/TemplateComponent/Code/
Removing MudBlazor.Docs/Pages/Components/TextField/Code/
Removing MudBlazor.Docs/Pages/Components/TimePicker/Code/
Removing MudBlazor.Docs/Pages/Components/Timeline/Code/
Removing MudBlazor.Docs/Pages/Components/ToggleGroup/Code/
Removing MudBlazor.Docs/Pages/Components/ToggleIconButton/Code/
Removing MudBlazor.Docs/Pages/Components/ToolBar/Code/
Removing MudBlazor.Docs/Pages/Components/Tooltip/Code/
Removing MudBlazor.Docs/Pages/Components/TreeView/Code/
Removing MudBlazor.Docs/Pages/Components/Typography/Code/
Removing MudBlazor.Docs/Pages/Customization/Globals/Code/
Removing MudBlazor.Docs/Pages/Customization/Theming/Code/
Removing MudBlazor.Docs/Pages/Features/Colors/Code/
Removing MudBlazor.Docs/Pages/Features/Converters/Code/
Removing MudBlazor.Docs/Pages/Features/Elevation/Code/
Removing MudBlazor.Docs/Pages/Features/Masking/Code/
Removing MudBlazor.Docs/Pages/Features/RTLLanguages/Code/
Removing MudBlazor.Docs/Pages/Getting Started/Installation/Code/
Removing MudBlazor.Docs/Pages/Getting Started/Layouts/Code/
Removing MudBlazor.Docs/Pages/Getting Started/Usage/Code/
Removing MudBlazor.Docs/Pages/Getting Started/Wireframes/Content/Code/
Removing MudBlazor.Docs/Pages/Getting Started/Wireframes/Wireframes/Code/
Removing MudBlazor.Docs/bin/
Removing MudBlazor.Docs/obj/
Removing MudBlazor.Docs/wwwroot/MudBlazorDocs.min.css
Removing MudBlazor.Examples.Data/bin/
Removing MudBlazor.Examples.Data/obj/
Removing MudBlazor.SourceCodeGenerator/
Removing MudBlazor.SourceGenerator/bin/
Removing MudBlazor.SourceGenerator/obj/
Removing MudBlazor.UnitTests.Docs/Generated/ApiDocsTests.generated.cs
Removing MudBlazor.UnitTests.Docs/Generated/ExampleDocsTests.generated.cs
Removing MudBlazor.UnitTests.Docs/bin/
Removing MudBlazor.UnitTests.Docs/obj/
Removing MudBlazor.UnitTests.Shared/bin/
Removing MudBlazor.UnitTests.Shared/obj/
Removing MudBlazor.UnitTests.Viewer/MudBlazor.UnitTests.Viewer.csproj.user
Removing MudBlazor.UnitTests.Viewer/bin/
Removing MudBlazor.UnitTests.Viewer/obj/
Removing MudBlazor.UnitTests/bin/
Removing MudBlazor.UnitTests/obj/
Removing MudBlazor/bin/
Removing MudBlazor/obj/
Removing MudBlazor/wwwroot/MudBlazor.min.css
Removing MudBlazor/wwwroot/MudBlazor.min.js
PM> dotnet build
  Determining projects to restore...
  All projects are up-to-date for restore.
  MudBlazor.SourceGenerator -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.SourceGenerator\bin\Debug\netstandard2.0\MudBlazor.SourceGenerator.dll
  MudBlazor.Analyzers -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Analyzers\bin\Debug\netstandard2.0\MudBlazor.Analyzers.dll
  wwwroot/MudBlazor.min.js Created
  wwwroot/MudBlazor.min.js UpToDate
  MudBlazor.Examples.Data -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Examples.Data\bin\Debug\net9.0\MudBlazor.Examples.Data.dll
  MudBlazor -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor\bin\Debug\net8.0\MudBlazor.dll
  MudBlazor -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor\bin\Debug\net9.0\MudBlazor.dll
  Generating Docs and Tests
MSBUILD : error MSB1009: Project file does not exist. [C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs\MudBlazor.Docs.csproj]
  Switch: 'C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs.Compiler/MudBlazor.Docs.Compiler.csproj'
  
  The build failed. Fix the build errors and run again.
C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs\MudBlazor.Docs.csproj(25,5): error MSB3073: The command "dotnet run --configuration Release --project 'C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs.Compiler/MudBlazor.Docs.Compiler.csproj'" exited with code 1.
  MudBlazor.UnitTests.Shared -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.UnitTests.Shared\bin\Debug\net9.0\MudBlazor.UnitTests.Shared.dll
  MudBlazor.Docs.Compiler -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs.Compiler\bin\Debug\net9.0\MudBlazor.Docs.Compiler.dll
  MudBlazor.Analyzers.TestComponents -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Analyzers.TestComponents\bin\Debug\net9.0\MudBlazor.Analyzers.TestComponents.dll
  MudBlazor.UnitTests.Viewer -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.UnitTests.Viewer\bin\Debug\net9.0\MudBlazor.UnitTests.Viewer.dll
  MudBlazor.UnitTests.Viewer (Blazor output) -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.UnitTests.Viewer\bin\Debug\net9.0\wwwroot
  MudBlazor.UnitTests -> C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.UnitTests\bin\Debug\net9.0\MudBlazor.UnitTests.dll

Build FAILED.

MSBUILD : error MSB1009: Project file does not exist. [C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs\MudBlazor.Docs.csproj]
C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs\MudBlazor.Docs.csproj(25,5): error MSB3073: The command "dotnet run --configuration Release --project 'C:\Users\danch\Source\Repos\MudBlazor\src\MudBlazor.Docs.Compiler/MudBlazor.Docs.Compiler.csproj'" exited with code 1.
    0 Warning(s)
    2 Error(s)

Time Elapsed 00:00:42.90
PM> 
```

--- a/src/MudBlazor.Docs/MudBlazor.Docs.csproj
+++ b/src/MudBlazor.Docs/MudBlazor.Docs.csproj
@@ -22,7 +22,7 @@
   <Target Name="CompileDocs" AfterTargets="ResolveProjectReferences" Condition="'$(DesignTimeBuild)' != 'true'">
     <!--Command-line for the code generator-->
     <Message Text="Generating Docs and Tests" Importance="high" />
-    <Exec Command="dotnet run --configuration Release --project '$(SolutionDir)MudBlazor.Docs.Compiler/MudBlazor.Docs.Compiler.csproj'" />
+    <Exec Command="dotnet run --configuration Release --project &quot;$(SolutionDir)MudBlazor.Docs.Compiler/MudBlazor.Docs.Compiler.csproj&quot;" />
   </Target>
 
   <!--Add Content that is being generated as part of the build cycle-->

